corona-warn-app/cwa-website
# Shorten title for FAQ link text preventing text overflow out of box (closes #849)

This PR mitigates the issue "Text is going out of the box in #de" #849 on https://www.coronawarn.app/de/.

---
**BEFORE CHANGE**

Near the bottom of the page there is a link to the FAQs. The text "Häufig gestellte Fragen zur Corona-Warn-App" overflowed the box and even the browser window as the window was decreased in width. See the following screen shot:

![FAQ box overflow](https://user-images.githubusercontent.com/66998419/107266026-3fc38d00-6a45-11eb-9544-c998d0226692.png)

---

**AFTER CHANGE**

The text "Häufig gestellte Fragen zur Corona-Warn-App" has been changed to "Antworten in den FAQs" which is a shorter length than the previous text. The problem no longer surfaces with the new text. As the browser width is reduced, the text stays within the box and the browser window. Finally the text is moved down so it still fits even when the browser window has been set to the minimum width allowed for the page. See below for the screen shots after the change:

![FAQ box text packed](https://user-images.githubusercontent.com/66998419/107266494-e871ec80-6a45-11eb-8ef2-81236da9b943.png)

![FAQ box minimum size](https://user-images.githubusercontent.com/66998419/107266506-ec057380-6a45-11eb-914f-1ca045c9d340.png)

---
## Steps to verify

1. Open https://www.coronawarn.app/de/ in web browser 
2. Scroll down to "Mehr Fragen?"
3. Reduce the browser to minimum width and observe that the text "Antworten in den FAQs" and arrow to the right of it stays inside the box and inside the browser window at all times as the width of the browser window is increased.

## Verification results

Confirmed with following browsers under Microsoft Windows using http://localhost:8000/de/ and `npm start`

- Google Chrome 88.0.4324.150
- Mozilla Firefox 85.0.1
- Microsoft Edge 88.0.705.63
